ui/src/pages/View.js

```diff
@@ -179,12 +212,30 @@ class View extends Component {
 
   getSexValue() {
     if (this.state.person.sex == null) {
-      return '';
+      return null;
     } else {
       return this.props.intl.formatMessage(
           SEX_VALUE_MESSAGES[this.state.person.sex]);
     }
   }
+  
+  getSourceDateValue() {
+    if (this.state.person.source_date == null) {
+      return null;
+    } else {
+      return new Date(this.state.person.source_date).toLocaleString();
```

Comment:
We might want to consider doing this formatting server-side and returning a string from the FE API instead of a raw timestamp. If someone selects a language in PF that's different from what their browser thinks they want, I think it'd be better if our date formatting matched the rest of what we serve them.